### DISCUSSION TITLE
Errors in Domoticz

### DISCUSSION THREAD

**Discussion by ElMyggo** [UtilitechAS/amsreader-firmware#411] [Q&A] [answered]
Finally im up and running with POW-K and Domoticz!
Think i find a error, i configure domoticz for volt L1, volt L2, volt L3, Ampere 3Phase and Power.
Works except the Power until the total kwh value is send out (same as the displayed value on meter)
then the power started to work but need to change to energy Wh instead of total counter.
Note that screenshot is not taken same minutes... some diff in value

Maybe this should be in Beta but same error in 2.1.17

![domoticz4](https://user-images.githubusercontent.com/111691693/215175985-9aa62f6c-2d84-426a-a6c0-f7ea1c9420b4.jpg)
![domoticz5](https://user-images.githubusercontent.com/111691693/215176057-605e1f8e-11ee-4654-9add-42157abd3b7c.jpg)
![domoticz6](https://user-images.githubusercontent.com/111691693/215176774-2e11ab94-a397-4c9e-a7dd-fc5afd95a0ff.jpg)

**Comment by gskjold** [comment]
I'm not sure I understand, what is the error exactly?

**Comment by ElMyggo** [comment]
Ok i try to explain a little better, not so good to do this in english ;-)

I update to 2.20 now and the value below import meter counting up from 0 and there is **no** mqtt output from Electricity IDX
Volt IDX and ampere IDX sending out ok.

But every hour when Kamstrup meter send L2 values then the mqtt **start to send** imported Watt and total counter value.

So i get no imported Watt send to domoticz before the L2 values is come in from meter.


Hope i have explain little better

![ams1](https://user-images.githubusercontent.com/111691693/215272268-8cc255e2-b6bf-4d70-9293-822f225765c1.jpg)

**Comment by ElMyggo** [comment]
Maybe this is not an error....
This maybe occur only on Kamstrup because it only send total counter value every hour.. other value sends every 10 second.

**Comment by gskjold** [comment, marked as answer]
I am not a Domoticz user myself and the option in this firmware was provided to us by a user. As far as I could tell from him, this is how it had to be to make sure you did not get any invalid data into the counter. This is also stated in the Wiki: https://github.com/UtilitechAS/amsreader-firmware/wiki/MQTT-configuration#domoticz

**Reply by ElMyggo** [reply]
OK, we skip this "error" because this is no error just a behavior :-)
